src/main/java/twilightforest/entity/EntityTFChainBlock.java

```diff
@@ -244,6 +245,11 @@ public void tick() {
 		}
 	}
 
+	@Override
+	protected void registerData() {
+		// TODO: What does this do?
```

Comment:
for registering datawatcher entries, which this entity has none. the todo can be removed

Reply:
In that case, I'm going to move into the abstract base class and it can be overriden if needed.

Reply:
Actually, scratch that, that might be unclear. I'll just remove the todo.